test/ctim/test_helpers/core.cljc

```diff
@@ -27,9 +23,11 @@
     (binding [cs/*recursion-limit* n]
       (t))))
 
+; this function used to override gen/vector with
+; a function with different behavior, which broke
+; internal generator logic in test.check 0.10.0.
 (defn fixture-fast-gen [t]
```

Comment:
```suggestion
(defn ^{:deprecated "1.0.16"} fixture-fast-gen [t]
```

Reply:
`fixture-fast-gen` is still used in just about every `ctim` test namespace (it just does nothing now). I think deprecating it can wait because it would mean deleting all the references and this PR is big and tangled enough.

Reply:
Opened an issue to do later https://github.com/threatgrid/ctim/issues/270